{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -44,7 +44,7 @@ dev-start: ## Primary make command for devs, spins up containers
 	docker-compose -f docker/docker-compose.yml --project-name $(PROJECT) up -d --no-recreate
 
 dev-stop: ## Spin down active containers
-	docker-compose -f docker/docker-compose.yml --project-name {{ cookiecutter.repo_name }} down
+	docker-compose -f docker/docker-compose.yml --project-name $(PROJECT) down
```

Comment:
If we're going to keep `repo_name` and `module_name` as separate things, then we need to quote `$(PROJECT)` to avoid errors due to whitespace. Better, we should guarantee that the `repo_name` doesn't have any whitespace in the pre-gen hook, since GitHub actively renames the repo from `example repo` to `example-repo` anyway

Reply:
In other words, let's either merge `module_name` and `repo_name` or add another check for `repo_name` to the pre-gen hook to make sure it doesn't have whitespace.